contributor-guide/modules/ROOT/pages/getting-involved.adoc

```diff
@@ -1,4 +1,58 @@
 = Getting Involved
 :navtitle: Getting Involved
 
-*Content under construction.*
\ No newline at end of file
+There are many ways to get involved with Boost, up to and including submitting your own library. In roughly ascending order of time and work commitment, here are the ways you can contribute to Boost:
+
+== 1. Join the Boost Users Mailing List
+
+The Boost mailing list is a platform for discussions and communications related to the Boost Libraries. The mailing list serves several key purposes:
+
+[disc]
+* *Announcements*: The mailing list is a channel for updates, announcements about new releases, bug fixes, events, or any other relevant news.
+
+* *Collaboration*: The mailing list also serves as a platform for collaboration, fostering connections between developers and encouraging team-based projects.
+
+* *Discussion*: Developers discuss ideas, techniques, issues, and potential improvements related to Boost libraries.
+
+* *Support*: Newcomers or users encountering difficulties can seek help from the community. Experienced members provide solutions, advice, and share their experiences.
+
+If you do join, follow the guidelines and rules of the mailing list, as it helps in maintaining a respectful and productive environment.
+
+To subscribe to the Users Mailing List go https://lists.boost.org/mailman/listinfo.cgi/boost-users[Boost Users mailing list].
+
+There are other mailing lists you might want to join, such as for Developers in general, and some project-specific lists. Visit https://www.boost.org/community/groups.html[Boost Mailing Lists] for more details
+
+== 2. Reporting Issues and Suggesting Improvements
+
+To formally report an issue, click the *New Issue* button on this GitHub page: https://github.com/cppalliance/site-docs/issues[cppalliance/site-docs]. If your issue is pertinent to just one library, there is a list of issues for just that individual library you can add to. In this case, navigate to the repo for that library, and locate the *New Issue* button. 
```

Comment:
Users shouldn't report all issues to site-docs. They should report to the individual libraries. The second sentence talks about individual libraries but the first sentence is not clear about it just being about site-docs itself. We could just make it about libraries, as reporting an issue with site-docs is supposed to be a much more tangent use case.

Reply:
@alandefreitas good feedback - reworded the Reporting Issues section, and committed the changes.